pkgs/tools/misc/turbo/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, fetchFromGitHub, buildGoModule, protobuf, protoc-gen-go, protoc-gen-go-grpc }:
+let
+  src = fetchFromGitHub {
+    owner = "vercel";
+    repo = "turbo";
+    rev = "69681a3396c377f5bc663bf3f942a62034694f29";
```

Comment:
If using commit hashes here, the version should be `unstable-YYYY-MM-DD`.

Reply:
I used a tag instead